java-frontend/src/main/java/org/sonar/java/resolve/SemanticModel.java

```diff
@@ -129,6 +130,10 @@ public Symbol getEnclosingClass(Tree tree) {
     return getEnv(tree).enclosingClass;
   }
 
+  public Type getClassType(String fullyQualifiedName) {
```

Comment:
I did not spot the unit test of that one. Especially to check how this one reacts with edgecases. (null or empty string, non existing class, etc.)